src/passport-saml/saml.ts

```diff
@@ -485,7 +501,7 @@ class SAML {
     Object.keys(additionalParameters).forEach((k) => {
       samlMessage[k] = additionalParameters[k];
     });
-    if (this.options.privateKey) {
+    if (this.options.privateKey != null) {
```

Comment:
I'm not sure what this change does. Does it force you to explicitly set `privateKey` to `null` to not use it?
I would agree, but you'd probably want to add this information to the readme.

Reply:
This changes the check from `falsy` to checking for `null` or `undefined`, which both are `falsy`. So, while it does changes things a little, it is now explicitly checking for the absence of a value, not just a value that is `falsy`. If it were `privateKey !== null`, that would be an explicit check for `null`.